src/renderer/components/+network-services/service-details.tsx

```diff
@@ -61,7 +61,15 @@ export class ServiceDetails extends React.Component<Props> {
         )}
 
         <DrawerItem name={<Trans>Ports</Trans>}>
-          <ServicePorts service={service}/>
+          <div>
+            {
+              service.getPorts().map((port) => {
+                return(
+                  <ServicePortComponent service={service} port={port}/>
+                );
+              })
```

Comment:
This can be shortened easily.
```suggestion
              service.getPorts().map((port) => (
                <ServicePortComponent service={service} port={port}/>
              ))
```